app/controllers/collaborations_controller.rb

```diff
@@ -1,2 +1,20 @@
 class CollaborationsController < ApplicationController
+  def create
```

Comment:
`CollaborationsController` and some of the other controllers don't use `before_action :authenticate_user!`. Is this right?